src/Scrim.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Scrim from './Scrim';
+
+const setup = (propOverrides) => {
+  const props = Object.assign({
+    // default props
+  }, propOverrides);
+
+  const wrapper = shallow(<Scrim {...props} />);
+
+  return {
+    props,
+    wrapper,
+  };
+};
+
+describe('<Scrim />', () => {
+
+  it('Render with correct class', () => {
+    const scrim = setup({ className: 'test' });
+    expect(scrim.wrapper.find('.test')).toHaveLength(1);
+  });
+
+  it('Component can be clickable', (done) => {
+    const scrim = setup({ onClick: done() });
+    scrim.wrapper.simulate('click');
```

Comment:
seems like this test is missing an expectation, could we add one here?

Reply:
i think i might be misunderstanding what done is doing here actually. Just never seen a test without an expectation! Do you think one should be added or is this fine?